doc/modules/neural_networks_supervised.rst

```diff
@@ -148,13 +162,21 @@ value is `1` represents the assigned classes of that sample::
     >>> clf.predict([0., 0.])
     array([[0, 1]])
 
+Following plot shows varying decision function depending on the value of alpha.
```

Comment:
I don't see explanation of alpha anywhere in the doc, I can either add a line or two about alpha in this doc itself or remove the image, thoughts?